docs/wiki/UtilitiesConversionScript.md

```diff
@@ -0,0 +1,54 @@
+# Convert bicep files into ARM helper script
+
+At the time of writing bicep is still in a beta phase. For this reason, some people may want to wait for bicep's _General Availability_ and prefer to use ARM Templates meanwhile.
+For those scenarios, a script is supplied with the CARML library, leveraging the bicep toolkit translator/compiler, to support the conversion from CARML bicep modules to ARM Templates.
+This page documents the conversion utility and how to use it.
+
+---
+
+### _Navigation_
+
+- [Convert bicep files into ARM helper script](#convert-bicep-files-into-arm-helper-script)
```

Comment:
Remove the first to lines. No need to list the page header and navigation itself in the navigation

Reply:
can't do. They reappear when I save the file

Reply:
done